src/systems/ec/ec-resources/icons/ui/if_gear_298781.svg

```diff
@@ -0,0 +1,5 @@
+<!-- Generated by IcoMoon.io -->
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="28" height="32" viewBox="0 0 28 32">
+<title>if_gear_298781</title>
```

Comment:
Can we rename this `cog` as per https://webgate.ec.europa.eu/CITnet/jira/browse/INNO-953 or more preferably `gear`?